pep-0678.rst

```diff
@@ -229,6 +232,49 @@ proposed ``__note__`` semantics, but this would be rarely and inconsistently
 applicable.
 
 
+Augment the ``raise`` statement
+-------------------------------
+One discussion proposed ``raise Exception() with "note contents"`` .
+This would be a strictly additional feature over adding the attribute and modifying
+the traceback-formatting code, and we do not expect ``__note__`` to be so
+frequently-used as to justify new syntax.
```

Comment:
I think it’s not so much the frequency of use of note, but the way that note will typically be used. Catch-and-reraise-with-note is not the use case that motivated this pep.

Same comment re the decorator idea. Both of these could be added, now or later, but are not necessarily in the scope of this pep.

Reply:
We don't have new syntax for setting the traceback either, but there's a `with_traceback(tb)` method. I could imagine adding a `.with_note(nt)` method, but only if there's a use case that would want to use this rather than assignment to `__note__` (for tracebacks, assignment to `__traceback__` works too, but there's a common idiom where the method is useful).

Reply:
> I think it’s not so much the frequency of use of note, but the way that note will typically be used. Catch-and-reraise-with-note is not the use case that motivated this pep.

On a related point - I'd reword the second sentence of the abstract slightly. It is currently "Because further information may be available when the exception is caught and re-raised...".  Maybe remove the 'and reraised' because that doesn't describe the ExceptionGroups use case.

Reply:
>  I'd reword the second sentence of the abstract slightly. [...]

Makes sense. (Although the reader might wonder why bother storing the note on the exception rather than just passing it along to whatever you pass the exception to. The answer is exactly ExceptionGroup, so maybe mention that explicitly?)